packages/lesswrong/server/resolvers/reviewPredictionResolvers.ts

```diff
@@ -0,0 +1,42 @@
+import { accessFilterMultiple } from "@/lib/utils/schemaUtils";
+import { getPostMarketInfo } from "@/lib/collections/posts/annualReviewMarkets";
+import moment from "moment";
+import gql from "graphql-tag";
+
+export const reviewPredictionGraphQLTypeDefs = gql`
+  extend type Query {
+    reviewPredictionPosts(year: Int!, limit: Int = 50): [Post!]!
+  }
+`;
+
+async function fetchCandidatePosts(year: number, context: ResolverContext): Promise<DbPost[]> {
+  const start = moment.utc(`${year}-01-01`).toDate();
+  const end = moment.utc(`${year + 1}-01-01`).toDate();
+  return context.Posts.find({
+    postedAt: { $gte: start, $lt: end },
+    status: 2,
+    draft: false,
+    shortform: false,
+    unlisted: false,
+    isEvent: false,
+    manifoldReviewMarketId: { $exists: true },
+  }, { limit: 2000 }).fetch();
```

Comment:
surely we don't need 2000 here if we're always only fetching 50?  (Also we're not doing any ordering/sorting?)

Reply:
Am I being dumb? I think that we can't order by the manifold market probability, so we need get all the ones with markets, then sort after the DB fetch

Reply:
It looks like I failed to actually leave the following comment immediately afterwards:

Oh, I see, you're doing the ordering in code.  Hm.  If you've tested it and the performance is tolerable, then fine, though I don't love it.  (It'd be great if it could've just been a join, but I guess we do have the problem of not always having real-time results for the manifold info.  Still...)